packages/mui-base/src/useSelect/useSelect.ts

```diff
@@ -4,6 +4,7 @@ import {
   unstable_useForkRef as useForkRef,
   unstable_useId as useId,
   unstable_useEnhancedEffect as useEnhancedEffect,
+  visuallyHidden as visuallyHiddenStyle,
```

Comment:
**Off-topic**

One day, it would be great to move this to its own folder

```suggestion
import { visuallyHidden } from '@mui/utils/visuallyHidden',
```

per #35840